src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -134,6 +137,24 @@ public Path getAndroidNDKPath() throws IOException {
         return resolvePath(configuration.getAndroidNdkPath(),"Fatal error, could not install Android NDK ");
     }
 
+    /**
+     * Checks that the required Android packages are present, else proceeds to
+     * install them
+     *
+     * @param androidSdk The path to the Android SDK
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    public void checkAndroidPackages(String androidSdk) throws IOException, InterruptedException {
+        if (Stream.of(ANDROID_SDK_PACKAGES)
+                .filter(s -> !s.contains("ndk"))
+                .map(s -> Path.of(androidSdk, s.split(";")))
+                .anyMatch(p -> !Files.exists(p))) {
+            Logger.logInfo("Some required Android packages were not found, they will be downloaded.");
```

Comment:
It would be good to print the missing packages here, so that the developer (or we) can verify

Reply:
Fixed